vocab_files/attribute_concepts/transect-start-point.ttl

```diff
@@ -0,0 +1,19 @@
+PREFIX dcterms: <http://purl.org/dc/terms/>
+PREFIX rdfs: <http://www.w3.org/2000/01/rdf-schema#>
+PREFIX schema: <https://schema.org/>
+PREFIX skos: <http://www.w3.org/2004/02/skos/core#>
+PREFIX tern: <https://w3id.org/tern/ontologies/tern/>
+PREFIX xsd: <http://www.w3.org/2001/XMLSchema#>
+
+<https://linked.data.gov.au/def/nrm/9beaf2dc-c53d-4e6e-b699-c1786f04f1f6>
+    a skos:Concept ;
+    dcterms:source """McCallum K, Laws M, Bignall J, O’Neill S, Sparrow B. (unpublished draft) ‘Coarse Woody Debris module’ in ….. (eds)
```

Comment:
there are more "s in the souce

Reply:
three quotes are used when the text is more than one line